README.md

```diff
@@ -168,6 +168,7 @@ services:
 |PASSWORD           |               | Both PASSWORD and USERNAME must be set in order to use Basic authorization
 |PASSWD_PATH        | /etc/nginx/.htpasswd | Alternate auth support (don't combine with USERNAME/PASSWORD) Bind-mount a custom path to `/etc/nginx/.htpasswd`
 |ADD_HEADER         | Not set       | Useful for tagging routes in your infrastructure.
+|SERVER_NAMES_HASH_SIZE         | 32       | Variable conditionning maximum size of server name. Set it to 64/128/... if nginx fails to start with `could not build server_names_hash, you should increase server_names_hash_bucket_size` error message.
```

Comment:
I'd split this into 2 parts....


First for the `SERVER_NAMES_HASH_SIZE`'s Notes field, try this: "Configures size of server name hash. Suggested values include 64, 128, 256, etc."

=================================

_This next bit should probably be under a troubleshooting section_

## Troubleshooting
* Failing to start
    * "could not build server_names_hash"
    If nginx fails to start with `could not build server_names_hash, you should increase server_names_hash_bucket_size` error message, you might need to configure environment variable: [`SERVER_NAMES_HASH_SIZE`](#arguments).

Reply:
Good idea!